ext/storage_sources/azure_store/azure_storage_source.cpp

```diff
@@ -37,17 +36,24 @@
 #include <vector>
 
 #include "azure_connection.h"
+#include "azure_log_system.h"
 #include "wt_internal.h"
 
+#include <azure/core/diagnostics/logger.hpp>
+
+using namespace Azure::Core::Diagnostics;
+
 struct azure_file_system;
 struct azure_file_handle;
 struct azure_store {
     WT_STORAGE_SOURCE store;
     WT_EXTENSION_API *wt_api;
+    std::unique_ptr<azure_log_system> log;
```

Comment:
How many `azure_store` instances can there be for a program? If there are multiple, should we consider having a `shared_ptr` here? Or do we want one `azure_log_system` per store?

Reply:
I'll double check this!

Reply:
There should only be 1 instance of `azure_store`

Reply:
There should be 1 log system per store and theres one store per program

Reply:
Cool, thanks.